src/Exceptions/HttpFailedDependency.php

```diff
@@ -8,4 +8,3 @@ class HttpFailedDependency extends HttpException {
     public const DESCRIPTION = "Means that the method could not be performed on the resource because the requested action depended on another action and that action failed.";
     public const HREF = "https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/424";
 }
-
```

Comment:
Removed unix line ending - ALWAYS have a unix line ending, it can cause very weird issues.

Just use https://marketplace.visualstudio.com/items?itemName=jkiviluoto.tws you can use this to add them automatically. It also removes trailing whitespaces